language/German/strings.po

```diff
@@ -721,23 +724,21 @@ msgstr "Zurück zum vorherigen Menü"
 msgid "File Manager"
 msgstr "Dateimanager"
 
-#: src/frontend/mame/ui/filemngr.cpp:140
-#: src/frontend/mame/ui/confswitch.cpp:119
-#: src/frontend/mame/ui/inputmap.cpp:580
-#: src/frontend/mame/ui/inputtoggle.cpp:75
+#: src/frontend/mame/ui/filemngr.cpp:140 src/frontend/mame/ui/confswitch.cpp:119
+#: src/frontend/mame/ui/inputmap.cpp:580 src/frontend/mame/ui/inputtoggle.cpp:75
 #: src/frontend/mame/ui/analogipt.cpp:363
 #, c-format
 msgid "[root%1$s]"
-msgstr ""
+msgstr "[Wurzel%1$s]"
 
-#: src/frontend/mame/ui/filemngr.cpp:154
-#: src/frontend/mame/ui/confswitch.cpp:170 src/frontend/mame/ui/slotopt.cpp:204
+#: src/frontend/mame/ui/filemngr.cpp:154 src/frontend/mame/ui/confswitch.cpp:170
+#: src/frontend/mame/ui/slotopt.cpp:204
```

Comment:
This is another change re-wrapping the source locations.

Reply:
fixed